### ISSUE TITLE
Chrome Custom Tabs für FAQ nutzen

### ISSUE THREAD

**Issue by tobidwest** [corona-warn-app/cwa-app-android#512]
## Current Implementation
Versucht der*die Nutzer*in auf dem Startbildschirm der App die FAQ aufzurufen, so wird diese*r auf die Webseite der Bundesregierung weitergeleitet. Diese öffnet sich im Standardbrowser des Benutzers

## Suggested Enhancement
Statt die Seite im Browser zu öffnen, sollte diese über Webview dargestellt werden. Der Standard Chrome Custom Tabs erlaubt es, eine Basisnavigation leicht zu implementieren und anzupassen.

## Expected Benefits
Das Öffnen der FAQ innerhalb der App erweckt den Eindruck eines nativen Erlebnisses. So lässt sich die Benutzerführung, vor allem für ältere Menschen, signifikant vereinfachen. Der Wechsel zwischen mehreren Apps entfällt.

**Comment by uschindler** [comment]
I disagree with this. Links to external web sites should always be opened in the browser. For web sites intended to be displayed as part of the app (same design and no way to enter other URLs into an addressbar) are perfectly fine in the webview.
Google recommends to not open external pages inside apps.

**Comment by tobidwest** [comment]
> I disagree with this. Links to external web sites should always be opened in the browser. For web sites intended to be displayed as part of the app (same design and no way to enter other URLs into an addressbar) are perfectly fine in the webview.
> Google recommends to not open external pages inside apps.

I definitely agree in reference to a "classical" WebView. 
But [Chrome Custom Tabs](https://developer.chrome.com/multidevice/android/customtabs) is used for opening external websites when entering a URL is not needed and navigation on the website is (almost) not needed. As the button directs the user to www.bundesregierung.de and only a few ages are related to the Corona-Warn-App, this would be the better choice from my point of view.

**Comment by codingcatgirl** [comment]
I have to agree with @tobidwest. While opening external pages inside apps might be recommended by Google, the FAQ page about this app is imo not really an "external" page.

**Comment by SecJoe** [comment]
If Chrome is not installed on the device, i think it doesnt work. Like stated here you can implement an fallback instead: https://stackoverflow.com/questions/33063298/does-chrome-custom-tabs-require-the-user-to-download-the-chrome-app In this case you should fall back to the old implementation (not to WebView like in the link).

**Comment by friedger** [comment]
Adding an action button to call the help support line would make sense together with custom apps.

Chrome Custom Tabs is the wrong description, it should be just "Custom Tabs" because all popular browser support this feature.

**Comment by svengabr** [comment]
I have just checked the linked Jira issue.

Jira Ticket is flagged as:
Resolution: `Wont Fix`
Status: `Obsolete`

Developer comment:
> will be implemented in [EXPOSUREAPP-4916](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4916).
> 
> set to obsolete.

That linked ticket EXPOSUREAPP-4916 was implemented in 2.13
